### DISCUSSION TITLE
API to check if a material is disabled

### DISCUSSION THREAD

**Discussion by KabanFriends** [PaperMC/Paper#10566] [Ideas]
### Is your feature request related to a problem?

Minecraft 1.19.3 added Feature Flag system, which allows some items to be disabled. I think there should be a way for server plugins to check if a material is disabled.

### Describe the solution you'd like.

An API to check if a bukkit Material is disabled or not in the specified world would be helpful. It needs a world since worlds can have different datapacks.

### Describe alternatives you've considered.

I'm currently using NMS codes to find out if a block is disabled or not (`net.minecraft.world.level.block.Block#isEnabled`)

**Comment by Machine-Maker** [comment]
> It needs a world since worlds can have different datapacks.

Are you sure worlds can have their own datapacks? their is only one `datapacks` folder inside the main level folder.

**Comment by maestro-denery** [comment]
yeah, even tho feature flags are stored per ServerLevel, the main `ServerLevel#enabledFeatures()` refers to the `PrimaryLevelData` stored in `MinecraftServer`, so the world functionality is unnecessary

**Comment by maestro-denery** [comment]
Generally, I think this kind of API is needed not only for Materials, but for other content as well

**Comment by Machine-Maker** [comment]
Implemented in 925c3b9d2b4a4a1c0a6ea4dc2b06f4009cde02aa
